### PR TITLE
bump to wasmbus-rpc v0.13

### PR DESCRIPTION
**Pull request by connorsmith256**
This bumps all the examples to wasmbus-rpc v0.13

### CODE REVIEW

File: ngs/dogs-and-cats/Cargo.toml
Comment (stevelr) [inline ngs/dogs-and-cats/Cargo.toml:1]: bump the package version too

File: petclinic/actors/clinicapi/Cargo.toml
Comment (stevelr) [inline petclinic/actors/clinicapi/Cargo.toml:1]: bump package version

File: petclinic/actors/customers/Cargo.toml
Comment (stevelr) [inline petclinic/actors/customers/Cargo.toml:1]: bump package version

File: petclinic/actors/ui/Cargo.toml
Comment (stevelr) [inline petclinic/actors/ui/Cargo.toml:1]: bump package version

File: petclinic/actors/vets/Cargo.toml
Comment (stevelr) [inline petclinic/actors/vets/Cargo.toml:1]: bump package version

File: petclinic/actors/visits/Cargo.toml
Comment (stevelr) [inline petclinic/actors/visits/Cargo.toml:1]: bump package version

File: provider/factorial/Cargo.toml
Comment (stevelr) [inline provider/factorial/Cargo.toml:1]: bump package version

File: provider/fakepay/Cargo.toml
Comment (stevelr) [inline provider/fakepay/Cargo.toml:1]: bump package version

File: provider/python/Cargo.toml
Comment (stevelr) [inline provider/python/Cargo.toml:1]: bump package version

**Review by stevelr — CHANGES_REQUESTED** [review]

**Review by brooksmtownsend — APPROVED** [review]

**Comment by connorsmith256** [comment]
Merging under the assumption that the failing blobby integration tests are due to flakes (the error messages differ slightly and seem to be related to the HTTP server not being fully up)

I was able to compile blobby, link it to the published versions of http server and blobstore, and post/get data back

**Comment by connorsmith256** [comment]
Scratch that, 6th time was the charm 🙂

**Pull request merged**